.github/workflows/preview-docs.yml

```diff
@@ -43,10 +41,35 @@ jobs:
     if: |
       github.event.action == 'closed' &&
       github.event.pull_request.head.repo.full_name == github.repository
-    uses: Qiskit/gh-actions/.github/workflows/code-engine-cleanup.yml@main
```

Comment:
I didn't save this job to a reusable file because we only have the cleanup code one time; staging doesn't use cleanup because it simply overwrites the previous deploy.